Tribute.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <div id = "img-div">
+            <img id = "image" src="tribute-image1.jpg" alt="">
+            <div id = "img-caption">
+                Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+            </div>
+        </div>
+        <div>
+            <h4 id = "tribute-info">Here's a time line of Dr. Borlaug's life:</h4>
+        </div>
+        
+        <div id="dates">
+            <P>
+                <ul>
+                    <li>1914 - Born in Cresco, Iowa</li>
+                    <li>1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+                    <li>1945 - Discovers a way to grown wheat twice each season, doubling wheat yields</li>
+                    <li>1970 - receives the Nobel Peace Prize</li>
+                    <li>2009 - dies at the age of 95.</li>
+                </ul>
+
+            </P>
```

Comment:
- You might want to change your uppercase P tags to lowercase for elements and attribute names as this is the best practice